.storybook/preview-head.html

```diff
@@ -34,7 +34,7 @@
 
   .sbdocs-h4 {
     font-size: var(--text-fs-2-size) !important;
-    font-weight: var(--font-weight-bold) !important;
+    font-weight: var(--font-weight-medium) !important;
```

Comment:
is this the only one that should change to bold? i see another `bold` below here

Reply:
Thanks for pointing it out Walter.